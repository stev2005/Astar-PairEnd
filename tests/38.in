CCTGCCAGCCTTAAGCATTAAGAAACCGGCTAGGGTTGCATAATCGACCTCGCGGGTTAGCTGTGATCGAG
15
CGAGACTGACAGTGAG
AAGTGGCGCTACCCGTGATTACTCCCATCGCCTTCCCGAGCAAGGGCATCTT
GAAAATCGATCAATCGT
ATAGCGACCCAATTATTATATTTATTTCGGCGGCATTACCCCCCAGA
GTACTTGGTCT
AATGCTCCAGGCATGACTAAGACCTTAGCTTCAGTCGAGGTATT
GTCCTCTTTCTGCCTACAATTGCA
TTCTGCCCGG
ACATGTCTCCAGCACCGTCTGAACGCGT
AATGCGAAAACCATGACTTAAG
AGAGTGGGTGGCTACAACCCCCAGGTCGATAACATATCGGTATTC
TCGATTCTCTGGCGCGTGGGGCTCTGCGCGTAACCTAGGGGAATTGCG
AATCTACCCTTA
TATATATCGCCGAAA
GCGGATCA
