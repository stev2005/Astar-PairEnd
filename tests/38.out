7
25
6
24
5
20
11
4
12
9
22
24
4
7
2
